src/cdomains/arrayDomain.ml

```diff
@@ -39,15 +39,12 @@ let get_domain ~varAttr ~typAttr =
 
 let can_recover_from_top x = x <> TrivialDomain
 
-module type S =
+module type SMinusDomainAndRet =
```

Comment:
Why `SMinusDomainAndRet`? You are removing `get`, right? Also maybe just naming it `S0` or something like this is more transparent.